pkg/cmd/root/help_topic.go

```diff
@@ -91,44 +91,96 @@ var HelpTopics = map[string]map[string]string{
+
+		The --template flag requires a string argument in Go template syntax, and will only print those JSON values which match the query. In addition to the Go template functions in the standard library, the following functions can be used with this formatting directive:
+		- %[1]sautocolor%[1]s: like %[1]scolor%[1]s, but only emits color to terminals
+		- %[1]scolor <style> <input>%[1]s: colorize input using <https://github.com/mgutz/ansi>
+		- %[1]sjoin <sep> <list>%[1]s: joins values in the list using a separator
+		- %[1]spluck <field> <list>%[1]s: collects values of a field from all items in the input
+		- %[1]stablerow <fields>...%[1]s: aligns fields in output vertically as a table
+		- %[1]stablerender%[1]s: renders fields added by tablerow in place
+		- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
+		- %[1]stimefmt <format> <time>%[1]s: formats a timestamp using Go's Time.Format function
+		- %[1]struncate <length> <input>%[1]s: ensures input fits within length
+
+		To learn more about Go templates, see: <https://golang.org/pkg/text/template/>.
 		`, "`"),
 		"example": heredoc.Doc(`
-			# format issues as table
-			$ gh issue list --json number,title --template \
-			  '{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title}}{{end}}'
-
-			# format a pull request using multiple tables with headers
+			# default output format
+			$ gh pr list
+			Showing 23 of 23 open pull requests in cli/cli
+
+			#123  A helpful contribution          contribution-branch              about 1 day ago
+			#124  Improve the docs                docs-branch                      about 2 days ago
+			#125  An exciting new feature         feature-branch                   about 2 days ago
+
+
+			# adding the --json flag with a list of field names
+			$ gh pr list --json number,title,author
+			[
+			  {
+				"author": {
+				  "login": "monalisa"
+				},
+				"number": 123,
+				"title": "A helpful contribution"
+			  },
+			  {
+				"author": {
+				  "login": "codercat"
+				},
+				"number": 124,
+				"title": "Improve the docs"
+			  },
+			  {
+				"author": {
+				  "login": "cli-maintainer"
+				},
+				"number": 125,
+				"title": "An exciting new feature"
+			  }
+			]
+
+
+			# adding the --jq flag and filtering the output to select the second result
+			$ gh pr list --json number,title,author --jq '.[1]'
+			{"author": {"login": "codercat"},"number":124,"title": "Improve the docs"}
+
+
+			# adding the --template flag and modifying the display format
+			$ gh pr list --json number,title,headRefName,updatedAt --template \
+				'{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title .headRefName (timeago .updatedAt)}}{{end}}'
```

Comment:
This `--template` example outputs the exact format of regular `gh pr list`, which isn't very compelling as an example for using `--template`. How about changing something about this expression so that the final output is somehow different (and potentially more useful to some people) than regular `gh pr list`?

Reply:
I wondered about that. I think this is more helpful because it's already known, and you can see how to sort of reverse engineer it using templates. I personally find a small tweak or revealing how something is done is easier for me to pattern match against. What do you think?

Reply:
Well, it can stay like this if the goal was to demonstrate how one can reverse-engineer the default output. But, from the user's perspective, they might be scanning these examples and seeing how one simple `gh` invocation and one seemingly very complicated `gh` invocation both produce the same result.

I'll leave the final decision up to you. The example itself is indeed useful as a starting point for editing one's template.